src/Stream/test/BinderRabbitMQ.Test/RabbitBinderTests.cs

```diff
@@ -1516,133 +1531,144 @@ public void TestInternalHeadersNotPropagated()
     [Trait("Category", "SkipOnLinux")]
```

Comment:
Hm... this test isn't running at all in the CI environment

Reply:
It succeeds locally. Do I need to change something?

Reply:
I don't know why it is skipped on linux... we should try removing the attribute and seeing what happens, but that does not have to be in this PR

Reply:
Tracked at #995.